packages/react-components/src/components/VideoGallery/useVideoTileContextualMenuProps.ts

```diff
@@ -210,7 +210,7 @@ export const useVideoTileContextualMenuProps = (props: {
       return undefined;
     }
 
-    return { items, styles: {}, calloutProps: { preventDismissOnEvent } };
+    return { items, styles: {}, calloutProps: { preventDismissOnEvent }, shouldFocusOnContainer: false };
```

Comment:
Does this fix not do the oppotsite? reading the bug they are wanting the focus to move inside the menu when the button is actioned?

Reply:
Focus container is the behavior that focuses move on the container itself instead of move into the item under container - It would be easier to understand if you get a chance to check the attachment in the bug

Reply:
![image](https://github.com/user-attachments/assets/e7553bdd-c25e-4346-9cd1-fc8d1cfc7ba9)

Interesting enough, the other attachment in the bug shows it working correctly with no repro.